module_utils/network_lsr/argument_validator.py

```diff
@@ -465,6 +475,22 @@ def _validate_post(self, value, name, result):
 
 
 class ArgValidator_DictIP(ArgValidatorDict):
+    VALID_MODES = [
+        r"attempts:([1-9]\d*|0)$",
+        r"debug$",
+        r"edns0$",
+        r"ndots:([1-9]\d*|0)$",
+        r"no-check-names$",
+        r"no-reload$",
+        r"no-tld-query$",
+        r"rotate$",
+        r"single-request$",
+        r"single-request-reopen$",
+        r"timeout:([1-9]\d*|0)$",
+        r"trust-ad$",
+        r"use-vc$",
```

Comment:
Maybe add `^` to the beginning of the patterns

Reply:
updated